packages/@icon-magic/cli/src/index.ts

```diff
@@ -54,46 +59,70 @@ program
   });
 
 program
-  .command(
-    'distribute',
+  .command('distribute [inputPaths...]')
+  .description(
     'moves an icon from the source to the destination, applying types as specified'
   )
-  .option('-i, --inputPaths', 'path to the input directory of icons')
   .option(
-    '-o, --outputPath',
+    '-o, --outputPath [outputPath]',
     'path to the output directory where the generated assets are to be written to'
   )
   .option(
-    '-t, --type',
+    '-t, --type [type]',
     'type of icons format to handle, accepted types are svg|png|webp'
   )
   .option(
-    '-g, --groupByCategory',
+    '-g, --groupBy [groupBy]',
```

Comment:
change `groupByCategory` in distribute's `distributeByType`

Reply:
I haven't changed the interface.